SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -154,6 +155,10 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	TUniquePtr<SpatialLoadBalanceEnforcer> LoadBalanceEnforcer;
 	TUniquePtr<SpatialVirtualWorkerTranslator> VirtualWorkerTranslator;
 
+	// If this worker is authoritative over the translation, the manager will be instantiated.
+	TUniquePtr<SpatialVirtualWorkerTranslationManager> VirtualWorkerTranslationManager;
+	void InitializeVirtualWorkerTranslationManager();
```

Comment:
afaik there's a preferred order of declaration in classes: structs before members before functions (although to me grouping relevant things together seems logical)

Reply:
I moved it to the end of the members/start of the methods so that we get both code style and grouping! Winning!